code/modules/ameridian/ameridian_crystal.dm

```diff
@@ -89,17 +96,30 @@
 /obj/structure/ameridian_crystal/proc/spread()
 	var/list/turf_list = list()
 	for(var/turf/T in orange(spread_range, get_turf(src)))
-		if(locate(/obj/structure/ameridian_crystal) in T) // skip turfs that already have a crystal
+		if(!can_spread_to_turf(T))
 			continue
-		if(istype(T, /turf/simulated/open) || istype(T, /turf/space))
-			continue // Ignore turfs that are actually air
 		if(T.Enter(src)) // If we can "enter" on the tile then we can spread to it.
 			turf_list += T
 
 	if(turf_list.len)
 		var/turf/T = pick(turf_list)
 		new /obj/structure/ameridian_crystal(T) // We spread
 
+// Check the given turf to see if there is any special things that would prevent the spread
+/obj/structure/ameridian_crystal/proc/can_spread_to_turf(var/turf/T)
+	if(T)
+		if(istype(T, /turf/space)) // We can't spread in SPACE!
+			return FALSE
+		if(istype(T, /turf/simulated/open)) // Crystals can't float. Yet.
```

Comment:
You sure they dont just spread then fall down a level?
If it works like hivemind does that would be really cool to have a multy layered deep underground were shards fall form above